src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -2302,9 +2303,26 @@ private void UpdateAlreadyPlacedVisuals()
     /// <summary>
     ///   Updates the organelle model displayer to have the specified scene in it
     /// </summary>
-    private void UpdateOrganellePlaceHolderScene(SceneDisplayer organelleModel, string displayScene)
+    private void UpdateOrganellePlaceHolderScene(SceneDisplayer organelleModel,
+        string displayScene, OrganelleDefinition definition, int renderPriority)
     {
         organelleModel.Scene = displayScene;
+        if (organelleModel.CurrentlyShown != null)
+        {
+            GeometryInstance geometry;
+
+            // Fetch the actual model from the scene to get at the material we set the RenderPriority on
+            if (string.IsNullOrEmpty(definition.DisplaySceneModelPath))
+            {
+                geometry = (GeometryInstance)organelleModel.CurrentlyShown;
+            }
+            else
+            {
+                geometry = organelleModel.CurrentlyShown.GetNode<GeometryInstance>(definition.DisplaySceneModelPath);
+            }
+
+            geometry.MaterialOverride.RenderPriority = renderPriority;
+        }
```

Comment:
Unless it's inconvenient, I think this can be made into a helper method in SceneDisplayer and CurrentlyShown can be kept as private.

Reply:
I think this DisplaySceneModelPath handling is in pretty many places, so unifying those to a single helper method would also be nice.